### DISCUSSION TITLE
Ideas about future game improvement

### DISCUSSION THREAD

**Discussion by jvondermarck** [jvondermarck/dinosaur-exploder#78] [Q&A] [answered]
Hi !

If you have any ideas about how to improve the game, by adding some levels during the game play, make the game harder, and so on, just talk here about what your ideas :)

**Comment by Manvith-Shetty** [comment]
Hello @jvondermarck
I played the game and the working was good, But I feel there is an element of purpose missing in the game! There is no purpose in this game.
How about adding different levels from 1 to n and making the **difficulty** go from low to high by restricting the number of Dinasaur in each level. 
What do you think?
I have some more ideas too.......

**Reply by jvondermarck** [reply, marked as answer]
Hi @Manvith-Shetty you are totally right ! I would recommend you to create a feature issue on the repository, and work on it ! That's a great idea. Good luck. 😉
